lib/active_zuora/generator.rb

```diff
@@ -196,8 +196,7 @@ def add_extra_customizations
       customize 'RatePlanCharge' do
         include LazyAttr
         exclude_from_queries :overage_price, :included_units,
-          :discount_amount, :discount_percentage, :rollover_balance, :price
```

Comment:
Ah, I see.

I came to the same conclusion on the correct fix, but most likely a good amount of work.

However, I'm hesitant to go down the nasty path of forked gems. It seems like the active_zuora mainline would want to update its activesupport and activemodel dependency soon-ish. The VCR issue, on the other hand, is our problem. :)

I'm okay with relying on this fork to unblock your work on the operations-service consolidation. We'll want to revisit this decision again before the mainline changes drastically from our fork. Also, please add comments in the Gemfile and generator.rb (or the README, or both) explaining why the fork and its modifications are necessary.